float32 cell_size 				#meters
int16 row_size 					#number of cells
int16 column_size 				#number of cells
bool[] obstacle_grid
